pkgs/tools/misc/coreutils/default.nix

```diff
@@ -176,6 +178,18 @@ stdenv.mkDerivation rec {
     rm -r "$out/share"
   '';
 
+  passthru = {} // optionalAttrs (singleBinary != false) {
+    # everything in the single binary gets the same verdict, so we
+    # override _that case_ with verdicts from separate binaries.
+    #
+    # binlore only spots exec in runcon on some platforms (i.e., not
+    # darwin; I have a note that the behavior may need selinux?)
+    binlore = (binlore.synthesize coreutils ''
+      execer can bin/{chroot,env,install,nice,nohup,runcon,sort,split,stdbuf,timeout}
```

Comment:
I don't think it would work quite like this, but the underlying point is interesting:
- synthesize isn't collecting the underlying lore for the package passed, so this would produce an empty override
- lore is emitted with absolute paths, so we'd need something to relativize and remap it